lib/i18n/tasks/translators/base_translator.rb

```diff
@@ -102,6 +133,10 @@ def parse_value(untranslated, each_translated, opts)
         else
           untranslated
         end
+      rescue StopIteration => e
+        require 'pry'
```

Comment:
Left-over debugging code?

Reply:
Totally :)

Interestingly I was hitting it quite a bit using OpenAI when we had a batch of translations that was largely straight interpolations because it would return a mismatch of values, eg, we'd send 50 translation strings and get 49 back. Even specifically prompting to give the matching amount back, I guess ChatGPT is just bad at counting.